doc/whats_new/v0.21.rst

```diff
@@ -116,6 +116,11 @@ Support for Python 3.4 and below has been officially dropped.
   :issue:`12334` by :user:`Emmanuel Arias <eamanu@eamanu.com>`,
   `Joel Nothman`_ and `Andreas Müller`_
 
+  -|API| The parameter ``labels`` is deprecated in version 0.21 and will be
```

Comment:
a blank after ``-``, it'd be better if you can take some time to double check your PR :)

Reply:
Good idea, I have been depending on flake8 to catch those errors, but realize flake8 is not useful for this file.